tasks/eslint.js

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+  grunt.config('eslint', {
+    app: {
+      eslintrc: '.eslintrc',
+      src: ['Gruntfile.js', 'tasks/*.js', 'src/*.js']
+    },
+    test: {
+      eslintrc: '.eslintrc',
+      src: [
+        'tests/*.js'
+      ]
```

Comment:
I see the following warning when running `grunt lint`:

```bash
➜  fxa-password-strength-checker git:(issue-3-verbose) grunt lint
Running "eslint:app" (eslint) task

src/bloomdata_short_pwd.js
  0:0  warning  File ignored because of your .eslintignore file. Use --no-ignore to override

✖ 1 problem (0 errors, 1 warning)
```

We generally treat warnings like errors - they shouldn't be there.